### PR TITLE
[dashboard] add website slugs

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

<details>
<summary>Changes summary and walkthrough generated by Copilot</summary>

</details>

## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/utils.ts [hunk 1]
```diff
@@ -96,6 +96,7 @@ export function isWebsiteSlug(pathName: string) {
         "code-of-conduct",
         "contact",
         "docs",
+        "events",
         "features",
         "for",
         "gitpod-vs-github-codespaces",
```
Comment (ghostdevv) [inline components/dashboard/src/utils.ts:102]:
```suggestion
        "gitpod-vs-github-codespaces",
        "guides"
```

**Review by ghostdevv — COMMENTED** [review]
Few extra ones that may come in handy :pray:

**Review by selfcontained — APPROVED** [review]

File: components/dashboard/src/utils.ts [hunk 2]
```diff
@@ -96,6 +96,7 @@ export function isWebsiteSlug(pathName: string) {
         "code-of-conduct",
         "contact",
```
[the remark this answers is not in the document]
Comment (svenefftinge) [reply components/dashboard/src/utils.ts:97]: Thanks! Could you please create another PR, so we don't spend more time than necessary to roll this out.

**Pull request merged**